internal/jimmhttp/rebac_admin/identities_integration_test.go

```diff
@@ -103,14 +104,55 @@ func (s *identitiesSuite) TestIdentityGetGroups(c *gc.C) {
 		c.Assert(err, gc.IsNil)
 		token = *groups.Next.PageToken
 		for j := 0; j < len(groups.Data); j++ {
-			c.Assert(groups.Data[j].Name, gc.Equals, groupTags[i+j].Id())
+			c.Assert(groups.Data[j].Name, gc.Equals, fmt.Sprintf("group-test%d", i+j))
+			c.Assert(groupTags[j].Id(), gc.Matches, `\w*-\w*-\w*-\w*-\w*`)
 		}
 		if *groups.Next.PageToken == "" {
 			break
 		}
 	}
 }
 
+// TestIdentityGetGroupsWithMissingDbGroup tests the behaviour
+// of GetIdentityGroups when a tuple lingers in OpenFGA but the group
+// has been removed from the database.
+func (s *identitiesSuite) TestIdentityGetGroupsWithMissingDbGroup(c *gc.C) {
```

Comment:
TestGetIdentityGroupsWithDeletedDbGroup would be better (just because the method is named wrong)